.github/workflows/newman.yml

```diff
@@ -0,0 +1,81 @@
+name: Postman Tests
+
+on:
+  push:
+
+jobs:
+  postman_tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: centreon/tests/rest_api
+        
+    env:
+      CONTAINER_NAME: my_centreon_container
+      CENTREON_URL: http://localhost
+      CENTREON_IMAGE: docker.centreon.com/centreon/centreon-web-alma9:develop
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Login to registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}
+          username: ${{ secrets.DOCKER_REGISTRY_ID }}
+          password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+      
+      - name: Install pnpm
+        run: npm install pnpm -g
+
+      - name: Install Dependencies
+        run: pnpm install -f ./centreon/tests/rest_api/package.json
+
+      - name: Start Centreon Web Container
+        run: docker run --name $CONTAINER_NAME -d -p 4000:80 --health-cmd="curl -f $CENTREON_URL/ || exit 1" --health-interval=5s $CENTREON_IMAGE
+
+      - name: Wait for Centreon Web Container to Start
+        run: |
+          npx wait-on http://localhost:4000/ --timeout 100000
+        timeout-minutes: 5
+      
+      - name: List contents of node_modules
+        run: |
+          ls -lR node_modules

```

Comment:
```suggestion
```

Reply:
In fact i added this to make sure that my dependencies are properly installed, in other terms it's for debugging reasons.

Reply:
don't forget to get rid of it at the end then